articles/logic-apps/quickstart-create-logic-apps-with-visual-studio.md

```diff
@@ -203,10 +203,10 @@ then save your settings, for example:
 
    ![Deployment status output](./media/quickstart-create-logic-apps-with-visual-studio/logic-app-output-window.png)
 
-   If the connectors you selected require any input from you,
-   a **PowerShell window may pop-up** in the backgroud to prompt
-   for passwords or other secret keys as required. The deployment will
-   continue after you have entered these details.
+   If the connectors you selected need any input from you, 
+   a **PowerShell window might open in the background** and prompt 
```

Comment:
Please don't use bold, which is reserved for UX elements, for emphasis. There's actually no need to use emphasis here.